src/dodal/devices/pressure_jump_cell.py

```diff
@@ -0,0 +1,239 @@
+from ophyd_async.core import ConfigSignal, StandardReadable
+from ophyd_async.epics.signal import epics_signal_r, epics_signal_rw
+
+
+class PressureJumpCellPumpMode:
+    MANUAL = "Manual"
+    AUTO_PRESSURE = "Auto Pressure"
+    AUTO_POSITION = "Auto Position"
+
+
+class PressureJumpCellBusyStatus:
+    IDLE = "Idle"
+    BUSY = "Busy"
+
+
+class PressureJumpCellTimerState:
+    TIMEOUT = "TIMEOUT"
+    COUNTDOWN = "COUNTDOWN"
+
```

Comment:
could: other device files such as linkam (https://github.com/DiamondLightSource/dodal/blob/main/src/dodal/devices/linkam3.py) or hutch shutter (https://github.com/DiamondLightSource/dodal/blob/main/src/dodal/devices/hutch_shutter.py) have those classes explicitly extend enum

Reply:
Must: For these enums to work with the real PVs they must extend `str` and `enum`

Reply:
Improved the device structure in a930d192dd22f13bd011d76bedae5af776fef0a0 based on the components.

Reply:
Fixed enums in 1538a5b04e878c4a927c064a994dc051c1a82eb9